dallinger/command_line.py

```diff
@@ -313,71 +319,64 @@ def debug(verbose, bot):
                     'There was an error while starting the server. '
                     'Run with --verbose for details.'
                 )
-            if re.match('^.*? web.1 .*? Ready.$', line):
+            if re.match('^.*? \d+ workers$', line):
                 ready = True
                 break
 
-        epipe = 0
-        participant = None
         if ready:
-            host = config.get('host')
-            port = config.get('port')
-            public_interface = "{}:{}".format(host, port)
-            log("Server is running on {}. Press Ctrl+C to exit.".format(public_interface))
+            base_url = get_base_url()
+            log("Server is running on {}. Press Ctrl+C to exit.".format(base_url))
 
             # Call endpoint to launch the experiment
             log("Launching the experiment...")
             time.sleep(4)
-            requests.post('http://{}/launch'.format(public_interface))
+            requests.post('{}/launch'.format(base_url))
 
             # Monitor output from server process
             for line in iter(p.stdout.readline, ''):
                 if verbose:
                     sys.stdout.write(line)
 
                 # Open browser for new participants
-                match = re.search('New participant requested: (.*)$', line)
-                if match:
-                    url = match.group(1)
-                    if bot:
-                        log("Using a bot to simulate participant...")
-                        try:
-                            from dallinger_experiment import Bot
-                            participant = Bot(url)
-                            participant.run_experiment()
-                        except ImportError:
-                            log("This experiment does not have a bot.")
-                    else:
+                if not bot:
```

Comment:
I think this check is unnecessary -- this message is printed by the HotAirRecruiter, but above we set it to use the BotRecruiter when bot=True